### PR TITLE
Correction of indentation during file creation.

### PR DESCRIPTION
**Pull request by bzy107**
- [ ] Changes have been thoroughly tested to not break existing functionality.
- [ ] New functionality has been documented or existing documentation has been updated to reflect changes.
- [x] Visual changes are explained in the PR description using a screenshot/recording of before and after.

When creating resource files, the indentation is not consistent, and manual corrections are required each time. 
I have made a fix to address this issue.

I have conducted the verification as follows.

```bash
php artisan make:filament-resource Customer
```
|before|after|
|---|---|
|スクリーンショット 2023-11-07 2 10 51|スクリーンショット 2023-11-07 2 12 29|

```bash
php artisan make:filament-resource Customer --soft-deletes
```
|before|after|
|---|---|
|スクリーンショット 2023-11-07 2 14 46|スクリーンショット 2023-11-07 2 15 56|

Please confirm.

### CODE REVIEW

**Comment by zepfietje** [comment]
Do we need this in any other `make` commands too?

**Comment by bzy107** [comment]
No, I also tried the following commands, those are not a problem.

```bash
make:filament-page
make:filament-panel
make:filament-relation-manager
make:filament-widget
```

File: packages/support/src/Commands/Concerns/CanIndentStrings.php [hunk 1]
```diff
@@ -9,7 +9,7 @@ protected function indentString(string $string, int $level = 1): string
         return implode(
             PHP_EOL,
             array_map(
-                fn (string $line) => str_repeat('    ', $level) . "{$line}",
+                fn (string $line) => $line === '' ? '' : str_repeat('    ', $level) . "{$line}",
```
Comment (danharrin) [inline packages/support/src/Commands/Concerns/CanIndentStrings.php]:
```suggestion
                fn (string $line) => ($line !== '') ? (str_repeat('    ', $level) . "{$line}") : '',
```

**Comment by danharrin** [comment]
It's because all commands use this trait for indentation

**Pull request merged**